skyportal/handlers/api/analysis.py

```diff
@@ -808,6 +817,7 @@ async def post(self, analysis_resource_type, resource_id, analysis_service_id):
 
         # Now call the analysis service to start the analysis, using the `input` data
         # that we assembled above.
+        log(f'{get_app_base_url()}/{analysis.handled_by_url}/{analysis.token}')
```

Comment:
Maybe add a first few works here to help the logging?

Reply:
May be worth joining with urljoin to ensure all slashes get placed correctly.

Reply:
Fixed.